apps/wallet-dashboard/components/dialogs/staking/views/StakedInfo.tsx

```diff
@@ -38,6 +38,13 @@ export function StakedInfo({ validatorAddress, accountAddress }: StakedInfoProps
                     value={formatPercentageDisplay(apy, '--', isApyApproxZero)}
                     fullwidth
                 />
+                <KeyValueInfo
+                    keyText="Validator commission"
+                    value={commission}
+                    fullwidth
+                    tooltipText="The charge imposed by the validator for their staking services."
+                    tooltipPosition={TooltipPosition.Right}
```

Comment:
A horizontal sidebar is created when placed in the right position
```suggestion
                    tooltipPosition={TooltipPosition.Top}
```
![image](https://github.com/user-attachments/assets/2884320d-6bca-4ca8-9780-fcc7bde7afe6)